### ISSUE TITLE
https://vgc.sunoru.com/#/battles seems stale

### ISSUE THREAD

**Issue by sdeiley** [sunoru/vgc#1]
More a question than issue.

Found your [site](https://vgc.sunoru.com/#/) and I'm curious if what's deployed is up-to-date with this repo? Seems some functionality is missing, like script editor or filter/analytics functionality.

**Comment by sunoru** [comment]
Yes that's right. Sorry but I'm currently trying to upgrade the site to support S/V and too busy to finish. Those functions were there in last version. But only the main branch is deployed.

**Comment by sdeiley** [comment]
NW at all, thanks for clarifying. If I knew any TS I'd offer to help. Any rote or configuration type work I could offload?

**Comment by sunoru** [comment]
Thanks! I'm not sure what work can be done right now. If you are interested you can watch this repo, and maybe I'll post some issues when I need help
